app/controllers/api/v1/auth/passwords_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Api
+  module V1
+    module Auth
+      class PasswordsController < DeviseTokenAuth::PasswordsController
+        include ExceptionHandler
+        include ActAsApiRequest
+        include Api::V1::Concerns::AuthenticateUser
+
+        before_action :validate_redirect_url_param, only: []
```

Comment:
mmm, not because i'm overriding that config. Which it's actually inherited from DeviseTokenAuth::PasswordsController and as I overrided some actions there are callbacks that I do not want.